### PR TITLE
Vz mini pol assay

### PR DESCRIPTION
**Pull request by Starnite**

### CODE REVIEW

File: experiments/20161104-pol-assay-high-throughput/20161117-primer-design.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+### Primers for sequencing entire PB2 sequence
+
+Forward: 20-mer primer that ends 100bp upstream of end of pCI sequence
+Reverse: 20-mer primer that ends 100bp downstream of start of pCI sequence
+
+Forward: GCACCTATTGGTCTTACTGA
+Reverse: TTTCACAAATAAAGCATTTT
```
Comment (ericmjl) [inline experiments/20161104-pol-assay-high-throughput/20161117-primer-design.md]: Can you annotate the predicted melting temperature of the primer? It should ideally be roughly in the 55-65ºC range. If not, extend the primer until you hit that range.

File: goals/Nov-16.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+## Goals for Nov 17 - Nov 22
```
Comment (ericmjl) [inline goals/Nov-16.md:1]: I think you should remove this file from the branch. It's already present in a separate branch.

**Review by ericmjl — CHANGES_REQUESTED** [review]

File: experiments/20161104-pol-assay-high-throughput/20161117-primer-design.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
+### Primers for sequencing entire PB2 sequence
+
+Forward: ends 100bp upstream of end of pCI sequence
+Reverse: ends 100bp downstream of start of pCI sequence
+
+Forward: AGGCACCTATTGGTCTTACTGA (22-mer, Tm = 57C, %GC = 45)
+Reverse: CATCACAAATTTCACAAATAAAGCATTTT (29-mer, Tm = 55C, %GC = 24)
```
Comment (ericmjl) [inline experiments/20161104-pol-assay-high-throughput/20161117-primer-design.md:7]: Both primers are checked, they look good. I will order them in.

**Comment by ericmjl** [comment]
@Starnite: looks like there's an issue here - data compilation code in this branch is conflicting with data compilation code from other branch. Which one should be the one that is final? I suspect it's the other branch; you might have to revert changes from 6cdef56 and discard the changed files in this branch.

**Comment by Starnite** [comment]
Sorry, I was having issues with committing new changes on the mini-pol-assay branch. The final version should be the vz-data-analysis branch.

**Comment by ericmjl** [comment]
@Starnite in that case, be sure to do the following:

(1) In GitHub desktop, revert the change 6cdef56.
(2) After that, discard changes to the four conflicting files above.
(3) Push.

**Review by ericmjl — APPROVED** [review]

**Pull request merged**